buildSrc/src/main/kotlin/ProtocTask.kt

```diff
@@ -49,12 +57,12 @@ open class ProtocTask : AbstractExecTask<ProtocTask>(ProtocTask::class.java) {
 
     override fun exec() {
         executable = protoc.get()
-
         // Build CLI args
         val kotlinOut = listOfNotNull(
             kotlinPackage.orNull.let { "kotlin_package=$it" },
             logLevel.orNull.let { "log=$it" },
-            "json_use_proto_names=true",
+            jsonUseProtoNames.orElse(true).get().let{ "json_use_proto_names=$it" },
```

Comment:
may as well add a proper param for this. thoughts?

Reply:
`ProtocTask` is only used within the pbandk gradle files (it's not currently meant to be used by external projects), so we can make some assumptions. Such as, `json_use_proto_names` will always be `true`. But it doesn't hurt to declare these as proper params. Since you've already done the work, I'm good with leaving it in 👍